handlers/handler-governance/src/main/resources/config/lbc.handler.xml

```diff
@@ -0,0 +1,21 @@
+<!--
```

Comment:
Add test code to demo folder, and add assertions to test code to make sure your test is exactly executed.